homework-2/program5.java

```diff
@@ -0,0 +1,102 @@
+import java.awt.*;
+import java.io.*;
+import java.nio.file.FileAlreadyExistsException;
+import java.util.Scanner;
+
+public class program5
+{
+    public static void main(String args[])
+    {
+        try
+        {
+            testIOexception();
+        }
+        catch(IOException e)
+        {
+            System.out.println("Exception caught: "+e);
+        }
+        try
+        {
+            testNumberFormatException();
+        }
+        catch(NumberFormatException e)
+        {
+            System.out.println("Exception caught: "+e);
+        }
+        try
+        {
+            testArithmeticException();
+
+        }
+        catch(ArithmeticException e)
+        {
+            System.out.println("Exception caught: "+e);
```

Comment:
The exception message should point out the cause of the exception, or some other feedback to the user rather than just saying what has happened